share/revng/clang-format-style-file.yml

```diff
@@ -110,7 +110,7 @@
   PenaltyBreakBeforeFirstCallParameter: 800000,
   PenaltyBreakComment: 0,
   PenaltyBreakFirstLessLess: 120,
-  PenaltyBreakOpenParenthesis: 0,
+  PenaltyBreakOpenParenthesis: 800000,
```

Comment:
Uhm OK. Not sure how this will pan out in practice.

Reply:
> Not sure how this will pan out in practice.

What do you mean?

This will make clang-format a lot less likely to break nested function calls (as in, passing the result of one call as the first argument of the next), especially when function names are short-ish.

It has been quite frustrating in the past, since clang-format seemingly breaks legit looking code.

Reply:
:+1: